lib/targets/cuda/quda_fortran.cpp

```diff
@@ -0,0 +1,15 @@
+#include <quda.h>
```

Comment:
I guess this is an oversight and should be deleted?

Reply:
Possibly. I did not touch the fortran stuff at all.

Reply:
Removed this ass discussed in portability meeting